SETUP_SUBMITTER_CMF.md

```diff
@@ -0,0 +1,120 @@
+
+If you don't yet have a CMF set up, you can complete the instructions below as part of the steps titled "Worker host setup" and "Install software for jobs" as you create the CMF with these instructions: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-a-cmf.html, and then (If you're using EC2) create an AMI which can function as either your CMF worker or your Submitter.
+
+## Create a new Windows EC2 instance to install Unreal on (Optional)
+
+If you’re setting up on a brand new Windows EC2 Instance either as your submitter or as your CMF worker node, a g5.2xlarge instance with 200 GB of storage will likely be reasonable minimum:
+
+- Download the Epic Installer and install the latest version of Unreal (5.2 or higher is required)
+- NVidia Grid drivers - Follow windows instructions - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html#nvidia-GRID-driver
+
+The following steps: (“Install Build Tools”, "Environment Setup", "Deadline Software Installation", "Build the Plugin") should be followed whether you’re currently setting up your submitter machine or creating an instance/AMI to act as your worker node for Unreal jobs.
+
+## Install Build Tools
+
+The Unreal Submitter Plugin currently must be compiled locally.
+
+- Install Visual Studio 2022 or later using the Visual Studio Installer
+  - When installing, under “Workloads” select “Desktop development with C++”
+  - Under “Individual Components”, select a recent .NET Framework SDK (4.6.1 and 4.8.1 have been verified)
+
+## (Submitter only) Install Deadline Cloud Monitor
+
+Deadline Cloud Monitor is used to both manage your credentials for submitting jobs to Deadline Cloud as well as monitoring the status of your jobs.
+
+- Follow the instructions at https://docs.aws.amazon.com/deadline-cloud/latest/userguide/submitter.html#install-deadline-cloud-monitor
+- Sign in
+
+## Environment Setup
+
+- (If not already installed) Install a recent version of Python (3.12 has been verified)
+- Make sure your Environment Variables are set correctly. In System Environment Variables, your PATH should include:
+  - The path to your Python Installation (C:\Program Files\Python312 for example)
+  - The path to your Python Scripts folder (C:\Program Files\Python312\scripts for example)
+  - The path to your Unreal binaries (C:\Program Files\Epic Games\UE_5.4\Engine\Binaries\Win64)
+
+## Deadline Software Installation
+
+- (If this instance will be a worker agent) python -m pip install deadline-cloud-worker-agent
+- python -m pip install deadline-cloud-for-unreal-engine
+- clone or download deadline-cloud-for-unreal-engine
+
+## Build the Plugin
+
+Adjust the first two paths below based on where your installation of Unreal lives, and where you installed deadline-cloud-for-unreal-engine.
+
+From the Unreal Install Batchfiles Folder (Note the ‘package’ parameter can be any new directory, however you’ll want it to be called “UnrealDeadlineCloudService” later):
```

Comment:
I believe 'UnrealDeadlineCloudService' folder already exists if the User has already installed the Unreal Submitters. (If they haven't installed the Unreal Submitters, then they'll need to create this folder).

Reply:
So in these instructions I've got the plugin build/copy step happening first - so that folder wouldn't exist correct?  The Submitter installer has a couple issues we're planning to address soon (The default 5_2 folder being the main one)

Reply:
Yes, actually that makes sense (I see the Submitter install instructions are below instead).